ui-v2/tests/acceptance/steps/index-forwarding-steps.js

```diff
@@ -0,0 +1,11 @@
+import steps from './steps';
+
+// step definitions that are shared between features should be moved to the
+// tests/acceptance/steps/steps.js file
+
+export default function(assert) {
+  return steps(assert);
+  // .then('I should find a file', function() {
+  //   assert.ok(true, this.step);
+  // });
```

Comment:
Why is this one different?

Reply:
No idea :) probably while I was initially looking at the addon and looking into whether I could figure a way to remove these files. Will uncomment.